cppcache/test/ByteArrayFixture.cpp

```diff
@@ -20,7 +20,7 @@
 ::testing::AssertionResult ByteArrayFixture::assertByteArrayEqual(
     const char* /*expectedStr*/, const char* /*bytesStr*/, const char* expected,
     const apache::geode::client::ByteArray& bytes) {
-  // One would normally just use std::regex but gcc 4.4.7 is lacking.
+  // One would normally just use boost::regex but gcc 4.4.7 is lacking.
```

Comment:
How about just resolving the issue in the comment. We don't support GCC 4.4.7 anymore.

Reply:
Hmmm, impenetrable test code with a somewhat cryptic comment about a compiler we no longer use.  How about we just delete the comment and move on?

Reply:
Yup